common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -75,7 +75,8 @@ object Mappings {
     "subLocation" -> standardAnalysedString,
     "city" -> standardAnalysedString,
     "state" -> standardAnalysedString,
-    "country" -> standardAnalysedString
+    "country" -> standardAnalysedString,
+    "englishAnalysedCatchAll" -> sStemmerAnalysedString
```

Comment:
@jamesgorrie @akash1810 any thoughts on this? Are there other fields you've heard of having trouble due to mismatched field analyzer / query analyzer?

If not I wonder if it's better to just have another field for byline ("bylineEnglishAnalyzed or something)?